README/ReleaseNotes/v624/index.md

```diff
@@ -121,6 +121,11 @@ For details, see the RooFit tutorial [rf612_recoverFromInvalidParameters.C](http
 
 ## Networking Libraries
 
+- Enabled WLCG Bearer Tokens support in RDavix. Bearer tokens are part of WLCG capability-based infrastructure with capability-based scheme uses an infrastructure that describes what the bearer is allowed to do as opposed to who that bearer is. Token discovery procedure are developed according WLCG Bearer Token Discovery specification document (https://github.com/WLCG-AuthZ-WG/bearer-token-discovery/blob/master/specification.md). Short overview:
+   1. If the BEARER_TOKEN environment variable is set, then the value is taken to be the token contents.
+   2. If the BEARER_TOKEN_FILE environment variable is set, then its value is interpreted as a filename. The contents of the specified file are taken to be the token contents.
+   3. If the XDG_RUNTIME_DIR environment variable is set*, then take the token from the contents of $XDG_RUNTIME_DIR/bt_u$ID**.
```

Comment:
is the placement of the starts in `set*` vs `..ID**` intentional (one star followed by 2 stars)?